app/controllers/organizations_controller.rb

```diff
@@ -6,7 +6,8 @@ def index
   end
 
   def pending
-    @organizations = Organization.where(status: "pending")
+    @organizations = Organization.
+        where(status: "pending")
```

Comment:
Use 2 (not 4) spaces for indenting an expression in an assignment spanning multiple lines.